plugins.js

```diff
@@ -282,48 +284,44 @@ exports.onWindow = function (win) {
   });
 };
 
-exports.decorateMenu = function (tpl) {
-  let decorated = tpl;
+// decorates the base object by calling plugin[key]
+// for all the available plugins
+function decorateObject (base, key) {
+  let decorated = base;
   modules.forEach((plugin) => {
-    if (plugin.decorateMenu) {
-      const res = plugin.decorateMenu(decorated);
-      if (res) {
+    if (plugin[key]) {
+      const res = plugin[key](decorated);
+      if (res && 'object' === typeof res) {
         decorated = res;
       } else {
-        console.error('incompatible response type for `decorateMenu`');
+        notify('Plugin error!', `"${plugin._name}": invalid return type for \`${key}\``);
       }
     }
   });
+
   return decorated;
+}
+
+exports.decorateMenu = function (tpl) {
+  return decorateObject(tpl, 'decorateMenu');
 };
 
-exports.getDecoratedConfig = function () {
-  let decorated = config.getConfig();
-  modules.forEach((plugin) => {
-    if (plugin.decorateConfig) {
-      const res = plugin.decorateConfig(decorated);
-      if (res && 'object' === typeof res) {
-        decorated = res;
-      } else {
-        notify('Plugin error!', `"${plugin._name}": invalid return type for \`decorateConfig\``);
-      }
-    }
+exports.getDecoratedEnv = function () {
+  const baseEnv = Object.assign({}, process.env, {
+    LANG: app.getLocale(),
```

Comment:
`LANG: app.getLocale()` => `LANG: app.getLocale().replace('-', '_')`

Since `app.getLocale()` returns the locale with `-` (`en-US`) while `echo $LANG` in a shell is with underscore: `en_US`